en/docs/develop/components/api-proxies/api-proxy.md

```diff
@@ -41,6 +41,8 @@ Choreo runs a professional, enterprise-grade CI/CD process to deploy APIs to its
 
 Once you deploy the REST API to the development environment, it is then exposed by an API Gateway with API security turned on. Developers get a testable URL for their API which they can use to verify the functionality of the API they implemented.
 
+You may set up a separate environment to test the API proxy before pushing it to production. To deploy the API proxy to both the test environment and the development environment, enter the test environment URL as the sandbox endpoint and the API backend URL as the endpoint.
```

Comment:
`test environment and the` -> sandbox environment and the

Reply:
lets refer to the environment as Sandbox environment instead of test environment